internal/magic/binary.go

```diff
@@ -155,3 +153,39 @@ func Marc(raw []byte, limit uint32) bool {
 // | g   l   T   F    | 1                | ...      |
 var Glb = prefix([]byte("\x67\x6C\x54\x46\x02\x00\x00\x00"),
 	[]byte("\x67\x6C\x54\x46\x01\x00\x00\x00"))
+
+// TzIf matches a Time Zone Information Format (TZif) file.
+// See more: https://tools.ietf.org/id/draft-murchison-tzdist-tzif-00.html#rfc.section.3
+// Its header structure is shown below:
+// +---------------+---+
+// |  magic    (4) | <-+-- version (1)
+// +---------------+---+---------------------------------------+
+// |           [unused - reserved for future use] (15)         |
+// +---------------+---------------+---------------+-----------+
+// |  isutccnt (4) |  isstdcnt (4) |  leapcnt  (4) |
+// +---------------+---------------+---------------+
+// |  timecnt  (4) |  typecnt  (4) |  charcnt  (4) |
+func TzIf(raw []byte, limit uint32) bool {
+	// File is at least 44 bytes (header size).
+	if len(raw) < 44 {
+		return false
+	}
+
+	if !bytes.HasPrefix(raw, []byte("TZif")) {
+		return false
+	}
+
+	// Field "typecnt" MUST not be zero.
+	if bytes.Equal(raw[36:40], []byte{0x00}) {
+		return false
+	}
+
+	// Version has to be NUL (0x00), '2' (0x32) or '3' (0x33).
+	version := raw[4:5]
+	for _, validVersion := range [][]byte{{0x00}, {0x32}, {0x33}} {
+		if bytes.Equal(version, validVersion) {
+			return true
+		}
+	}
+	return false
```

Comment:
```suggestion
	return raw[4] == 0x00 || raw[4] == 0x32 || raw[4] == 0x33
```
You don't need to use `bytes.Equal` when comparing a single byte value. The suggested code is easier to read and faster to run.